README.md

```diff
@@ -42,6 +42,8 @@ Babbage runs independently. However, in order to run it locally in its publishin
 | ENABLE_CACHE                     | N                      | Switch to use (or not) the cache                                                                                  |
 | ENABLE_COVID19_FEATURE           |                        | Switch to use (or not) the covid feature                                                                          |
 | ENABLE_METRICS                   | N                      | Switch to collect (or not) metrics about cache expiry times                                                       |
+| ENABLE_LEGACY_CACHE_API          | N                      | Enable Cache API                                                                                                  |
```

Comment:
For consistency, the description should be "Enable legacy cache API". We need to call this properly, otherwise it gets confusing. Remember that Babbage already has a different caching feature.